CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### PaymentSheet
 * [FIXED][6434](https://github.com/stripe/stripe-android/pull/6434) Fixed an issue where the `Save this card for future payments` checkbox wasn't displayed in some cases even though it should have been.
+* [CHANGED][6471](https://github.com/stripe/stripe-android/pull/6471) Updated Google Pay button to match new brand guidelines.
```

Comment:
Nit: Can we move this to the first spot? I feel like our ordering should be `Added` > `Changed` > `Fixed`.